### PR TITLE
Add a web example

### PR DESCRIPTION
**Pull request by maasencioh**
It's going to be added a [Tonic](https://tonicdev.com/npm/ml-naivebayes) example

### CODE REVIEW

**Comment by maasencioh** [comment]
@JeffersonH44 @targos is this ok to you?

File: tonic.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var NaiveBayes = require('ml-naive-bayes');
```
Comment (targos) [inline tonic.js]: the package name on npm is `ml-naivebayes`

File: tonic.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+var nb = new NaiveBayes();
+
+// training data
+var cases = [[6,148,72,35,0,33.6,0.627,5],
+             [1.50,85,66.5,29,0,26.6,0.351,31],
+             [8,183,64,0,0,23.3,0.672,32],
+             [0.5,89,65.5,23,94,28.1,0.167,21],
+             [0,137,40,35,168,43.1,2.288,33]];
+var predictions = [1, 0, 1, 0, 1];
+nb.train(trainingSet, predictions);
```
Comment (targos) [inline tonic.js]: s/trainingSet/cases/

File: package.json [hunk 3]
```diff
@@ -8,9 +8,12 @@
...
-    "build": "gulp build"
+    "test": "mocha --require should --reporter mocha-better-spec-reporter --recursive"
   },
+  "engines": {
+    "node": ">=2.0.0"
+  },
+  "tonicExampleFilename" : "./tonic.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/mljs/naiveBayes.git"
```
Comment (targos) [inline package.json]: while you are at it, can you update the URL here ?
should be `https://github.com/mljs/naive-bayes.git`

Comment (targos) [inline package.json]: same for "bugs" and "homepage" fields

**Comment by targos** [comment]
a few nits but LGTM

**Pull request merged**